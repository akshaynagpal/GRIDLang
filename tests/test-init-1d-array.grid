int gameloop(){
    int[4] x;
    x = [1,2,3,4];
    print("Success.1d array intialized.");
    return 0;
}