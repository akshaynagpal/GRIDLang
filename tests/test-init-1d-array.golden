Success.1d array intialized.